src/pages/App/index.tsx

```diff
@@ -158,4 +157,4 @@ function App(): JSX.Element {
   );
 }
 
-export default config.nodeEnv === 'development' ? hot(App) : App;
+export default process.env.NODE_ENV === 'development' ? hot(App) : App;
```

Comment:
why revert back

Reply:
I think I messed up with merging or pulling somewhere along the line... I did not change it back myself :)))